### PR TITLE
 Height visualisation , closes #138

### LINKED ISSUE #138: Visualisation of Height
The height of a potential partner is a very important information for most users. I want to visualise this on the profile page to "see" it right away. Have a look at https://max-interactive.com/projects/people/ by @xamiam to get an idea of the concept. The current_user and the profile person stand next to each other in front of a visible 2,00 m and 1,50 m mark. The basic idea:

Bildschirmfoto 2024-03-06 um 14 48 42

It should be placed above the fold on a mobile phone. But it can be small. 

I will send @briankariuki additional files which GitHub doesn't allow me to attach to this issue.

### PR DESCRIPTION
**Pull request by MICHAELMUNAVU83**
- Hello @wintermeyer  ,On the profile page , we show a visualization for the user's height and their preferred potential partner height. The SVGs got tricky so i decided to use CSS , it is now as such .
Screenshot 2024-04-03 at 15 44 19
Screenshot 2024-04-03 at 15 44 43

### CODE REVIEW

File: lib/animina_web/components/animina_components.ex [hunk 1]
```diff
@@ -170,4 +170,116 @@ defmodule AniminaWeb.AniminaComponents do
...
+  Flag card loading.
+
+  ## Examples
+
+    <.flag_card_loading />
+  """
+
+  def height_visualization(assigns) do
+    ~H"""
+    <%= if @current_user.gender == "male"  || @current_user== "female" do %>
```
Comment (wintermeyer) [inline lib/animina_web/components/animina_components.ex]: This does not cover the case when the profile itself is diverse. We want to exclude that case for now too.

File: lib/animina_web/live/profile_live.ex [hunk 2]
```diff
@@ -12,6 +12,12 @@ defmodule AniminaWeb.ProfileLive do
 
   @impl true
   def mount(%{"username" => username}, %{"language" => language} = _session, socket) do
+    average_potential_partner_height =
```
Comment (wintermeyer) [inline lib/animina_web/live/profile_live.ex]: What do you mean by "average" here?

**Review by wintermeyer — CHANGES_REQUESTED** [review]

File: lib/animina_web/live/profile_live.ex (hunk 2 above)
Comment (MICHAELMUNAVU83) [reply lib/animina_web/live/profile_live.ex]: Here , I am getting the average between the maximum height and the minimum height that the user selected

File: lib/animina_web/components/animina_components.ex (hunk 1 above)
Comment (MICHAELMUNAVU83) [reply lib/animina_web/components/animina_components.ex]: Just to get it right @wintermeyer  , we should display the height visualization when the profile is only male or female . if not , if the profile is diverse , how should we approach it

File: lib/animina_web/live/profile_live.ex (hunk 2 above)
Comment (wintermeyer) [reply lib/animina_web/live/profile_live.ex]: Why do we need that information? The current_user has only access to profiles which fit his settings.

File: lib/animina_web/components/animina_components.ex (hunk 1 above)
Comment (wintermeyer) [reply lib/animina_web/components/animina_components.ex]: We only display this visualisation if all participants in it are either male or female.

File: lib/animina_web/live/profile_live.ex (hunk 2 above)
Comment (MICHAELMUNAVU83) [reply lib/animina_web/live/profile_live.ex]: I understand now , the height comparison should be between  the profile being viewed and the current user looking at the profile

Comment (wintermeyer) [reply lib/animina_web/live/profile_live.ex]: > I understand now , the height comparison should be between the profile being viewed and the current user looking at the profile

Yes! It should be a quick way to "feel" the height of the profile compared to myself.

Comment (MICHAELMUNAVU83) [reply lib/animina_web/live/profile_live.ex]: @wintermeyer  , thanks for this explanation , I have made tha changes for this , the current user on the left and the profile being viewed on the right .

**Pull request merged**